namespaceResourceQuotaResizing/nsQuotaResizing1_singleContainer/nsQuotaResizing_singleContainer.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: quotatest1
+---
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: quotatest1
+  namespace: quotatest1
+spec:
+  hard:
+    limits.cpu: "100m"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cpu-resize-up
+  namespace: quotatest1
```

Comment:
nit: Can we give these names that are a little more descriptive of the scenario, something like "quotatest-single-container" instead of quotatest1, "quota-test-mixed-executability" instead of quotatest2, "quotatest-up-and-down" instead of quotatest3, etc.? I think that would help when we are looking at these namespaces when they are deployed to know what scenario they represent.

Reply:
Thanks! Updated the NS name